lint/action.yaml

```diff
@@ -25,27 +22,7 @@ runs:
       if: inputs.checkout-repo == 'true'
       with:
         fetch-depth: 0
-    - name: Setup tools
-      uses: open-turo/action-setup-tools@v1
-    # TODO: Configure .pypi credentials using pypi-token
-    - name: Install poetry
-      if: hashFiles('poetry.lock') != ''
-      shell: bash
-      run: ${GITHUB_ACTION_PATH}/../lib/poetry-install.sh
-    - name: Make sure poetry uses the current python version
-      if: hashFiles('poetry.lock') != ''
-      shell: bash
-      run: |
-        # Use the version that pyenv gives as python version pyenv version-name
-        if ! command -v pyenv &>/dev/null; then
-          echo "pyenv not found, skipping"
-          exit 0
-        fi
-        PYTHON_VERSION_SET=$(pyenv version-name)
-        poetry env use $PYTHON_VERSION_SET
-    - name: Install pip dependencies
-      if: hashFiles('poetry.lock') == ''
-      shell: bash
-      run: pip install -e .
-    - name: Pre-commit
-      uses: open-turo/action-pre-commit@v1
+    - name: Setup python
+      uses: actions/setup-python@v5
+    - uses: pre-commit/action@v3.0.1
```

Comment:
Open turo uses that action as well. I think we can keep it as is since quite a bit of the gain in time comes from not using `open-turo/action-pre-commit` which calls again action setup tools etc

Reply:
I think instead we should fix `open-turo/action-pre-commit` if that is where the bottleneck is

Reply:
just simplifying it here since actions-python can work by default with pre-commit which is a python package. 

Not sure if this would be as simple for other languages though. 

In the end it's doing the same thing as here: https://github.com/open-turo/action-pre-commit/blob/main/action.yaml#L101